moduleframework/common.py

```diff
@@ -719,3 +718,24 @@ def get_module_type_base():
     if parent not in get_backend_list():
         raise ModuleFrameworkException("As parent is allowed just base type: %s" % get_backend_list)
     return parent
+
+
+def get_docker_file(dir_name="../"):
+    """
+    Function returns full path to dockerfile.
+    :param dir_name: dir_name, where should be Dockerfile located
+    :return: full_path to Dockerfile
+    """
+    fromenv = os.environ.get("DOCKERFILE")
```

Comment:
this should be documented

Reply:
http://meta-test-family.readthedocs.io/en/latest/user_guide/environment_variables.html

Reply:
Documented.